### ISSUE TITLE
dropdown: disabled dropdown should still be focusable and reveal options

### ISSUE THREAD

**Issue by nikkimk** [patternfly/patternfly-elements#2669]
## Description of the issue

[WAI-ARIA recommends](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#focusabilityofdisabledcontrols) elements of a larger composite widget remain focusable.

### Impacted component(s)

- `pf-dropdown`

### Steps to reproduce

1. Go to [Dropdown documentation -> Disabled dropdown](https://patternflyelements.org/components/dropdown/#disabled-dropdown-1)
2. Try to tab to the dropdown and open it.

### Expected behavior

A disabled dropdown should still open but have all options disabled.

**Comment by bennypowers** [comment]
the disabled dropdown at http://v4-archive.patternfly.org/v4/components/dropdown#disabled-toggles is not focusable. we can't diverge from upstream. a11y issues on patternfly should be filed there.

**Comment by bennypowers** [comment]
> the disabled dropdown at http://v4-archive.patternfly.org/v4/components/dropdown#disabled-toggles is not focusable. we can't diverge from upstream. a11y issues on patternfly should be filed there.

As discussed in chat, Nikki verified with pf team that her solution should be adopted upstream
